website/screens/components/nav-tabs/code/NavTabsCodePage.tsx

```diff
@@ -139,7 +139,7 @@ const sections = [
               anchor element which allows it to be used in any React based
               router. For each API is different so here are some examples for{" "}
               <DxcLink
-                href="https://reactrouter.com/docs/en/v6/hooks/use-href"
+                href="https://reactrouter.com/en/main/start/examples"
```

Comment:
This link goes to a page where there is only a link to a GitHub repo. Should we link directly to that repo?
![image](https://user-images.githubusercontent.com/67998605/219649931-aef56d75-5600-4eec-adef-5cc7b9c2263b.png)